drivers/all_neighbors_system_dashboard/app/views/all_neighbors_system_dashboard/warehouse_reports/reports/_time_to_obtain_housing.haml

```diff
@@ -0,0 +1,111 @@
+- start_date = (DateTime.current - 3.years).beginning_of_month
```

Comment:
Is 3 years a fixed date from the spec, or should we move the start date and end dates into the report?